lib/ansiblelint/utils.py

```diff
@@ -355,7 +383,7 @@ def normalize_task_v2(task):
     # denormalize shell -> command conversion
     if '_uses_shell' in arguments:
         action = 'shell'
-        del(arguments['_uses_shell'])
+        del (arguments['_uses_shell'])
```

Comment:
Is there any reason for this change? Seems like it is completely against normal function calling conventions

Reply:
Sorry, my IDE just "optimized" the code this way. It slipped through. Shouldn't have commited that.